lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -254,6 +254,26 @@
         .course-container{
           border: 1px solid $border-color-l4;
           border-radius: 3px;
+          .xseries-container{
```

Comment:
NIT: add space between this and the above item.  

Also, could you add a comment above this line denoting this is a case of the course container? (// CASE: Xseries associated course) and rename this to .label-xseries-association {} instead of .xseries-container {} ? This would help clarify the styling below.